src/apis/auth.ts

```diff
@@ -1 +1,51 @@
-/* 로그인 관련 API 요청 */
\ No newline at end of file
+import { setAccessToken } from '@/utils/token';
+import axios from 'axios';
+
+interface ISocialLoginParams {
+  code: string | null;
+  redirect_url: string;
+}
+
+const getGoogleProcess = async ({ code, redirect_url }: ISocialLoginParams) => {
+  try {
+    const res = await axios.get(
+      `${process.env.NEXT_PUBLIC_BACKEND_URL}/api/login/google/process?code=${code}&redirect_url=${redirect_url}`,
+    );
+    const data = await res.data.message;
+    setAccessToken(data.accessToken);
+    return true;
+  } catch (err) {
+    alert(`구글 로그인 에러 발생 : ${err}`);
+    return false;
+  }
+};
```

Comment:
3개 다 동일한 함수 쓰는거 같은데
하나의 공통함수로 추상화하면 더 좋을거 같습니다!
```suggestion
const getAtuhProcess = async (
  provider: 'google' | 'naver' | 'kakao',
  { code, redirect_url }: ISocialLoginParams
): Promise<boolean> => {
  try {
    const { data } = await axios.get(
      `${process.env.NEXT_PUBLIC_BACKEND_URL}/api/login/${provider}/process?code=${code}&redirect_url=${redirect_url}`,
    );
    setAccessToken(data.message.accessToken);
    return true;
  } catch (err) {
    alert(`${provider} 로그인 에러 발생: ${err}`);
    return false;
  }
};
```